src/handlers/transformation/transform-csv-to-json.ts

```diff
@@ -0,0 +1,15 @@
+import { S3Event } from "aws-lambda";
+import * as AWS from "aws-sdk";
+import csv from "csvtojson";
+
+// Unsure how to test this function or indeed if it even needs testing given that it basically just using S3 and the csvtojson library. Feedback/Help welcome!
```

Comment:
I'd vote no -- it would seem like overkill to me.  This is for a utility we'll only be using temporarily, and which will be run by us devs...